.github/workflows/ci.yaml

```diff
@@ -19,8 +19,20 @@ permissions:
   contents: read
 
 jobs:
-
-  lint:
+  lint-eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.2.2
+        with:
+          persist-credentials: false
+      - name: Install GitHub Actions formatter
+        run: npm install eslint-formatter-gha --no-save
```

Comment:
Do we need to pin the version? I.e., I could imagine that not all versions are compatible with all eslint versions.

Reply:
Sure, I can pin it just to be safe.